deployments/jwebbinar/image/default-home-contents/Desktop/DS9.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Version=1.0
+Type=Application
+Name=DS9
+Comment=
+Exec=/opt/conda/envs/jwebbinar/bin/ds9
```

Comment:
Not time to change it now,  but I'm wondering  what putting ds9 in jwebbinar achieves vs. keeping the common ds9 as a hidden kernel,   e.g. this isn't dedicated to jdaviz too.

Reply:
Do you have a suggestions as to have to have ds9 as a hidden kernel.

I think I'll file a ticket to revisit this later.

Reply:
How about this:

1. Leave ds9  where it is as a common kernel
2. Add pyds9 to common-env someplace so that it's installed in all environments for programmatic usage... assuming it does not cause conflicts.  (This is missing now.)
3. Basically keep your current  scheme for disabling the JupyterHub kernel after it's installed,  maybe wrap it in a common script like env-disable-kernel.

To the extent possible though the goal is to (a) solve it once and (b) still avoid conflicts.   If it's too hard,  just forget about it.

Reply:
That sounds like a good plan.  I'll file a ticket for this as well.  Thanks.